vividus-extension-selenium/src/main/java/org/vividus/ui/util/SearchAttributesConversionUtils.java

```diff
@@ -25,33 +25,37 @@
 
 import org.apache.commons.lang3.StringUtils;
 import org.vividus.selenium.LocatorFactory;
-import org.vividus.ui.web.action.search.ActionAttributeType;
-import org.vividus.ui.web.action.search.SearchAttributes;
-import org.vividus.ui.web.action.search.Visibility;
+import org.vividus.ui.action.search.IElementAction;
+import org.vividus.ui.action.search.IElementActionService;
+import org.vividus.ui.action.search.SearchAttributes;
+import org.vividus.ui.action.search.Visibility;
 
-public final class SearchAttributesConversionUtils
+public class SearchAttributesConversionUtils
 {
     private static final int SEARCH_VALUE_GROUP = 2;
     private static final int ATTRIBURTE_TYPE_GROUP = 1;
     private static final String EMPTY = "";
     private static final int ELEMENT_TYPE_GROUP = 3;
     private static final char CLOSING_BRACKET = ']';
-    private static final String LOCATOR_FORMAT = "(?:By\\.)?([a-zA-Z]+)\\((.+?)\\):?([a-zA-Z]*)?";
+    private static final String LOCATOR_FORMAT = "(?:By\\.)?([a-zA-Z]+)\\((.+?)\\):?([a-zA-Z]*)";
```

Comment:
What will be result of this locator parsing? `xpath(//a):`

Reply:
xpath: //a visibility: visible